requirements.txt

```diff
@@ -2,3 +2,4 @@ requests
 openai
 edge-tts
 rich
+EdgeGPT
```

Comment:
better to fix EdgeGPT version, it breaks sometimes
e.g.: https://github.com/yihong0618/xiaogpt/commit/81c33fbf8bbb200610fc75c282513f1ebc2bd378